.github/workflows/test.yaml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests
+# across different versions of node.
+# For more information see:
+#  https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Run Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version:
+          - 14.x
+          - 16.x
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
```

Comment:
any affinity to these node versions? I've been running most actions on 12.16.1

Reply:
I chose those because they are the [Active LTS and the current ](https://nodejs.org/en/about/releases/)versions, we can add the 12 series too!

Reply:
12 added ✔️

Reply:
thanks! I plan to at one point start migrating most projects to node 14, but until then... haha